src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -25,38 +26,41 @@
  * monitoring output and checking the exit code of the child process.
  */
 public class DefaultProcessRunner implements ProcessRunner {
-
-  private ProcessBuilder processBuilder;
-  private boolean async = false;
-  private ProcessOutputLineListener stdOutLineListener;
-  private ProcessOutputLineListener stdErrLineListener;
+  private boolean async;
+  private List<ProcessOutputLineListener> stdOutLineListeners;
+  private List<ProcessOutputLineListener> stdErrLineListeners;
   private ProcessExitListener exitListener;
 
+  private ProcessBuilder processBuilder = new ProcessBuilder();
   private Process process;
 
-  /**
-   * Create the process runner with a default process builder, with inheritIO enabled.
-   */
-  public DefaultProcessRunner() {
-    this(new ProcessBuilder());
-    processBuilder.inheritIO();
-  }
+  private Map<String, String> environment;
 
-  /**
-   * Create the process runner with the provided process builder.
-   */
-  public DefaultProcessRunner(ProcessBuilder processBuilder) {
-    this.processBuilder = processBuilder;
+  private DefaultProcessRunner(Builder builder) {
```

Comment:
Effectively, it's just another style that's doing the same thing. I'll switch to the style you're suggesting though because it looks like it's more common.